src/server.ts

```diff
@@ -1 +1,7 @@
-console.log('Hello Server!');
+import { readFile } from 'fs/promises';
+
+async function readPasswords() {
+  const db = await readFile('./src/db.json', 'utf-8');
+  console.log(db);
+}
+readPasswords();
```

Comment:
This should only contain the console.log refer to https://github.com/phlgr/vault